examples/AeroSharp.Examples/Maps/MapExampleData.cs

```diff
@@ -19,4 +20,35 @@ internal static class MapExampleData
         { "key9", "value9" },
         { "key10", "value10" }
     };
-}
+
+/*
+ * A nested map example
+ * {
+ *   { "key1" : {
+*        { "key21", 999.99 },
+ *       { "key22", 999.99 }
+ *   },
+ *   { "key2" : {
+ *       { "a", 5 },
+ *       { "c", 3 }
+ *   }
+ * }
+ */
+    public static readonly Dictionary<string, object> m3 = new()
```

Comment:
`m3` Feels a bit ambiguous.
```suggestion
    public static readonly Dictionary<string, object> NestedMap = new()
```

Reply:
It's the third map I made in testing this out and it just sounded cool - like a BMW M3